platform/android/detect.py

```diff
@@ -176,19 +185,35 @@ def configure(env: "SConsEnvironment"):
         )
     )
 
+    has_swappy = detect_swappy()
+    if not has_swappy:
+        print_warning(
+            "Swappy Frame Pacing not detected! It is strongly recommended you download it from https://github.com/darksylinc/godot-swappy/releases and extract it so that thirdparty/swappy-frame-pacing/arm64-v8a/libswappy_static.a can be found.\n"
+            + "Without Swappy, Godot apps on Android will inevitable suffer stutter and struggle to keep consistent 30/60/90/120 fps. Though Swappy cannot guarantee your app will be stutter-free, not having Swappy will guarantee there will be stutter even on the best phones and the most simple of scenes."
```

Comment:
Seems a bit alarmist.. This paragraph is basically asserting that up until now, all Godot Android apps / games have been riddled with stutters.. is that the case?

If not, I'd recommend removing this section, and maybe adding that swappy will improve game performance to the previous paragraph.

Reply:
> all Godot Android apps / games have been riddled with stutters.. is that the case?

Yes. It's really bad.